lib/public/views/lhcPeriods/format/formatDistinctLhcBeamEnergies.js

```diff
@@ -13,81 +14,50 @@
 
 import { h } from '/js/src/index.js';
 
-const distinctEnergiesProcessingConfiguration = { // FOR NOW HARDCODED
-    wellDefinedEnergyLevelToDisplay: {
-        450: 450,
-        6800: 6800,
-        7000: 7000,
-        2680: '5360/2',
-    },
-    acceptableEnergyMargin: 0.01,
-};
-
-/**
- * Get closest energy from given list to given lhcBeamEnergy
- * @param {number} lhcBeamEnergy lhc beam energy
- * @param {number[]} wellDefinedEnergyLevels list of energy levels
- * @returns {number} closest energy
- */
-export const getClosestDefinedEnergyLevel =
-    (lhcBeamEnergy, wellDefinedEnergyLevels) => {
-        const definedEnergyValues = Object.keys(wellDefinedEnergyLevels);
-        const closestEnergyLevel = definedEnergyValues.reduce(
-            (acc, current) => Math.abs(current - lhcBeamEnergy) < Math.abs(acc - lhcBeamEnergy) ? current : acc,
-            definedEnergyValues[0],
-        );
-        return closestEnergyLevel;
-    };
+const WELL_DEFINED_ENERGY_LEVELS = [450, 2680, 6800, 7000].sort((a, b) => a - b); // Sorted!
```

Comment:
You defined them as sorted, no need to sort it again
You can put the comment saying that the array is expected to be sorted though